src/Client/app/(admin)/sidebar/page.tsx

```
@@ -92,23 +77,14 @@ const AdminLayout = () => {
-              </div>
-              {!isCollapsed && (
-                <button
-                  onClick={handleThemeChange}
-                  className={`${isDarkMode ? 'text-white text-sm' : 'text-black text-sm'}`}
-                >
-                  Theme
-                </button>
-              )}
+            <div className="flex items-center gap-2 ml-2">
+              <ThemeSwitch />
+              {!isCollapsed && <button>Theme</button>}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)